.readme.sh

```diff
@@ -17,22 +17,86 @@ GO111MODULE=off go get -u github.com/myitcv/gobin
 export PATH=$(go env GOPATH)/bin:$PATH
 which gobin
 
+# ====================================
+# global examples
+
 # behind the scenes fix the version of gohack we install
-gobin github.com/rogpeppe/gohack@v0.0.1
+gobin github.com/rogpeppe/gohack@v1.0.0
 
 # block: gohack
 gobin github.com/rogpeppe/gohack
 
 # block: gohack latest
 gobin github.com/rogpeppe/gohack@latest
 
-# block: gohack v1.0.0-alpha.2
-gobin github.com/rogpeppe/gohack@v1.0.0-alpha.2
+# block: gohack v1.0.0
+gobin github.com/rogpeppe/gohack@v1.0.0
 
 # block: gohack print
-gobin -p github.com/rogpeppe/gohack@v1.0.0-alpha.2
+gobin -p github.com/rogpeppe/gohack@v1.0.0
 
 # block: gohack run
-gobin -r github.com/rogpeppe/gohack@v1.0.0-alpha.2 -help
+gobin -r github.com/rogpeppe/gohack@v1.0.0 -help
 assert "$? -eq 2" $LINENO
 
+# ====================================
+# main-module examples
+
+mkdir hello
+cd hello
+go mod init example.com/hello
+cat <<EOD > tools.go
+// +build tools
+
+package tools
+
+import (
+        _ "golang.org/x/tools/cmd/stringer"
+)
+EOD
+gofmt -w tools.go
+
+# block: module
+cat go.mod
+
+# block: tools
+cat tools.go
```

Comment:
this cat (and perhaps the go.mod one too) seem a bit verbose to me, given that the files were just written as-is.

Reply:
Yes I tend to use this as more of a copy-paste pattern between these sorts of scripts. Because it ensures, irrespective of how they are written above, that they will be formatted in the README